Generator/Sources/NeedleFramework/Utilities/Constants.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Copyright (c) 2018. Uber Technologies
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+/// The name of the NeedleFoundation module.
+let needleModuleName = "NeedleFoundation"
+
+/// The default value for waiting timeout.
```

Comment:
nit: add (in seconds) ?

Reply:
oh, wait.  where is this used?

Reply:
Sure. I'll add the unit. This is used in a bunch of places where we wait for the executor to finish.